modules/client-configuration/pages/bootstrap-repository.adoc

```diff
@@ -161,3 +161,41 @@ Automatic creation of a bootstrap repository might fail if you are using custom
 In this case, you need to create the repository manually.
 
 For more information about custom channels, see xref:administration:custom-channels.adoc[].
+
+
+
+=== Create Bootstrap Repository for {sll} 9
+
+////
+// https://bugzilla.suse.com/show_bug.cgi?id=1207809
+// https://github.com/SUSE/spacewalk/issues/20464
+// We can either add it here or to all "Register Clients" sections in
+// the clients-xxx.adoc files
+
+When creating the bootstrap repository for SUSE Liberty Linux 9-x86_64 the only file that is added is the salt bundle:
+
+Creating bootstrap repo for SUSE-LibertyLinux9-x86_64
+copy 'venv-salt-minion-3004-1.3.3.x86_64'
+Directory walk started
+Directory walk done - 1 packages
+Temporary output repo path: /srv/www/htdocs/pub/repositories/res/9/bootstrap.tmp/.repodata/
+Preparing sqlite DBs
+Pool started (with 5 workers)
+Pool finished
+////
+
+When creating the bootstrap repository for {sll} 9 the user must select the {salt} bundle contact method.
+
+
+* With the [command]``mgr-bootstrap`` command use the [option]``--force-bundle`` parameter, or
```

Comment:
I am not sure if the --force switch is really needed.
A current bootstrap script should use venv-salt-minion "by default".
If the bootstrap repo has it, it should be used. Also without this --force switch.

If somebody saw a problem, I would like to know more about it.

Reply:
Agreed, anything with `--force` looks sketchy. :D